src/etc/rc.newwanip

```diff
@@ -183,7 +183,12 @@ if (!empty($gre)) {
 	}
 }
 
-if (platform_booting()) {
+if (platform_booting() && strpos($interface_real, "ppp") !== 0) {
```

Comment:
This could also affect PPTP WAN and L2TP WANs, the same as PPPoE and PPP, so you should check for those as well. Maybe something like this:
```
in_array(substr($interface_real, 0, 3), array("ppp", "ppt", "l2t"))
```
Or a `preg_match()` with some variation of `"/^ppp|^l2tp|^pptp/"`